### PR TITLE
KAS-4647: Enable new mu-auth for testing

### PR DESCRIPTION
**Pull request by sergiofenoll**
https://kanselarij.atlassian.net/browse/KAS-4647

### CODE REVIEW

**Comment by sergiofenoll** [comment]
Jenkins builds:

- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/364/ :-1: 
- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/365/ :-1: 
- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/366/ :-1: :chart_with_upwards_trend: 
- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination_2/79/ CANCELLED
-  https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/380/ :-1: Some failures due to missing config for new internal review model & errors due to old ministers
- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/381/ :hourglass:

**Comment by ValenberghsSven** [comment]
Fixed an issue in shortlist service (bad error handling, but a ECONNRESET happened)
Fixed an issue in yggdrasil (`DELETE WHERE` without variables turned into a `SELECT DISTINCT WHERE` without variables which fails)
Bumped delta-notifier to v0.3.1
Bumped sparql-parser to v0.0.1 (current latest) to ensure we know what version we are testing
https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/369/

**Comment by ValenberghsSven** [comment]
Current "known" issues:
some ECONNRESETS where happening. Perhaps due to yggdrasil bug locking up the system?
mocklogin sessions are not always created or deleted or valid.

**Comment by ValenberghsSven** [comment]
merged the config changes to DEV already for easier deploy for testing

**Comment by ValenberghsSven** [comment]
`submissions-write` still contains minister-roles. I think they should be read-only now.

File: docker-compose.yml [hunk 1]
```diff
@@ -166,14 +165,16 @@ services:
     labels:
       - "logging=true"
   agenda-approve:
-    image: kanselarij/agenda-approve-service:5.11.2
+    image: kanselarij/agenda-approve-service:feature-wait-on-busy-timeouts
```
Comment (ValenberghsSven) [inline docker-compose.yml]: we probably don't want this. this disables concurrency check on agenda approving which would result in wrong VR numbers

Comment (ValenberghsSven) [reply docker-compose.yml]: reverted back to tag and lower timeout in development docker file.

File: docker-compose.yml [hunk 2]
```diff
@@ -238,9 +239,11 @@ services:
     labels:
       - "logging=true"
   case-documents-sync:
-    image: kanselarij/case-documents-sync-service:1.4.0
+    image: kanselarij/case-documents-sync-service:feature-max-body-size
     logging: *default-logging
     restart: always
+    environment:
+      MAX_BODY_SIZE: "500kb"
     labels:
```
Comment (ValenberghsSven) [inline docker-compose.yml]: MAX_BODY_SIZE is also declared in ENV properties in the PR `feature-max-body-size`
Do we need both?

**Review by tdn — APPROVED** [review]
let's go!

**Comment by tdn** [comment]
Accidentally merged this into development instead of acceptance, but pulled the changes from the branch directly afterward

**Pull request merged**